### PR TITLE
mrc-2614: Add function to get translated label and desc for plot types

### PR DESCRIPTION
**Pull request by r-ash**
This replaces #253 sorry for any confusion

Want to get review from science on the labels and description text. Please feel free to edit these files and push edits to this PR.

### CODE REVIEW

File: tests/testthat/test-input-time-series.R [hunk 1]
```diff
@@ -152,3 +152,19 @@ test_that("ART data throws error if dupe annual data or incomplete quarterly", {
...
+  expect_equal(ret, list(
+    list(
+      id = "art_total",
+      label = "ART count",
+      description = "Number on ART at the end of calendar year"
+    ),
+    list(
+      id = "art_child",
+      label = "ART paediatric",
+      description = "Proportion of total on-ART under age 15"
```
Comment (mwoodbri) [inline tests/testthat/test-input-time-series.R:167]:
```suggestion
      description = "Number of children (<15) on ART at the end of calendar year"
```
Not clear whether this should be "Proportion" or "Number" but test failure is due to mismatch with JSON

File: inst/traduire/en-translation.json [hunk 2]
```diff
@@ -233,6 +233,28 @@
...
+    "ART_TOTAL": "ART count",
+    "ART_TOTAL_DESC": "Number on ART at the end of calendar year",
+    "ART_ADULT_F": "ART female",
+    "ART_ADULT_F_DESC": "Number of women (15+) on ART at the end of calendar year",
+    "ART_ADULT_M": "ART male",
+    "ART_ADULT_M_DESC": "Number of men (15+) on ART at the end of calendar year",
+    "ART_ADULT": "ART adult",
+    "ART_ADULT_DESC": "Number of adults (15+) on ART at the end of calendar year",
+    "ART_CHILD": "ART paediatric",
+    "ART_CHILD_DESC": "Number of children (<15) on ART at the end of calendar year",
```
Comment (richfitz) [inline inst/traduire/en-translation.json:246]: Looks like we've got many more keys added here than in the fr and pt translations - what is our preferred workflow here? To add dummy keys, google translations, or leave as-is? I think that as-is the key will get reflected out into the application, which is fine for now. We could also perhaps more easily detect missing keys by looking to see what is present in the en set but missing from the others?

File: NAMESPACE [hunk 3]
```diff
@@ -8,6 +8,7 @@ export(anc_testing_prev_mf)
 export(artnum_mf)
 export(calculate_prevalence_art_coverage)
 export(calendar_quarter_labels)
+export(calendar_quarter_labels_short)
```
Comment (richfitz) [inline NAMESPACE:11]: Why is this exported here? unrelated change or needed in a dependent hintr pr?

**Review by richfitz — COMMENTED** [review]
two small queries here, code itself looks fine

File: inst/traduire/en-translation.json (hunk 2 above)
Comment (r-ash) [reply inst/traduire/en-translation.json:246]: Plan is at some point before estimates round (probably some time early Nov) do a diff from the last date we received translations to now. Then add those new keys to a spreadsheet for UNAIDS to translate.

Have previously to this point been adding translations from google or deepl so it is at least partially usable by people but after discussion the other day maybe this is not worth the effort as it is quite boring to add these placeholder translations.

File: NAMESPACE (hunk 3 above)
Comment (r-ash) [reply NAMESPACE:11]: Unrelated - the `@export` tag was added to docs at some point but they must not have regenerated

Comment (richfitz) [reply NAMESPACE:11]: ah, ok
